three.js/src/threex/arjs-markercontrols.js

```diff
@@ -89,11 +89,19 @@ const MarkerControls = function(context, object3d, parameters){
 MarkerControls.prototype = Object.create( ArBaseControls.prototype );
 MarkerControls.prototype.constructor = MarkerControls;
 
+
+//////////////////////////////////////////////////////////////////////////////
+//		dispose instance
+//////////////////////////////////////////////////////////////////////////////
 MarkerControls.prototype.dispose = function(){
-	this.context.removeMarker(this)
+	if( this.context && this.context.arController ) {
+		this.context.arController.removeEventListener('getMarker', this.onGetMarker);
+	}
 
-	// TODO remove the event listener if needed
-	// unloadMaker ???
+	this.context.removeMarker(this);
+	
+	this.object3d = null;
+	this.smoothMatrices = [];
```

Comment:
not sure if this is really needed. just wanted to be extra safe when i was debugging a memory leak.